src/herbivoresAndCarnivores.js

```diff
@@ -2,14 +2,42 @@
 
 class Animal {
   // write your code here
+  static alive = [];
+
+  health = 100;
+
+  constructor(name) {
+    this.name = name;
+    Animal.alive.push(this);
+  }
 }
 
 class Herbivore extends Animal {
   // write your code here
+  hidden = false;
```

Comment:
The `hidden` property is set to `false` by default, and the `hide` method sets it to `true`. Consider adding a method to unhide the herbivore or toggle the hidden state.